getting-started/other-frameworks/vitest/your-first-vitest-run.md

```
@@ -0,0 +1,131 @@
+
+See [Vitest reporters documentation](https://vitest.dev/guide/reporters#junit-reporter) for the full list of the JUnit reporter options.
+
+</details>
+
+<details>
+
+<summary>Update your .gitignore</summary>
+
+Add the converted reports directory and the JUnit XML file to your .gitignore to avoid pushing temporary generated reports to your repository.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)